libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -212,212 +243,184 @@ export class SkyDateRangePickerComponent
   @HostBinding('class.sky-margin-stacked-lg')
   public stacked = false;
 
-  @HostBinding('style.display')
-  public display: string | undefined;
-
-  public selectedCalculator: SkyDateRangeCalculator | undefined;
-
-  public readonly dateRangePickerId = `sky-date-range-picker-${uniqueId++}`;
+  /**
+   * Whether to require users to specify a start date.
+   * @deprecated Use the `required` directive or Angular's `Validators.required`
+   * on the form control to mark the date range picker as required.
```

Comment:
Added deprecation messages to both `startDateRequired` and `endDateRequired` inputs.